app.json

```diff
@@ -1,5 +1,5 @@
 {
-    "id": "c33fa1a6-68a7-491e-bf49-9d7b99671c48",
+    "id": "0a63d0d1-42a1-4951-80ed-023b89352473,
     "version": "3.0.0",
```

Comment:
Version should be `1.0.0` since this is a new app.

Reply:
you're right, it should ideally start with a new version since we already have a couple of releases on this repo, I went with the next release i.e. `3.0.0`